lib/public/views/Logs/Logs.js

```diff
@@ -50,14 +55,89 @@ export default class Overview extends Observable {
         this.notify();
     }
 
+    /**
+     * Create the log with the variables set in the model, handle errors appriopately
+     * @returns {undefined}
+     */
+    async createLog() {
+        const response = await fetchClient('api/logs', {
+            method: 'POST',
+            headers: {
+                Accept: 'application/json',
+                'Content-Type': 'application/json',
+            },
+            body: JSON.stringify({
+                title: this.getTitle(),
+                text: this.getText(),
+            }),
+        });
+
+        const result = await response.json();
+
+        if (result.data) {
+            const logId = await result.data.id;
+            this.fetchedFromCreation = true;
+            await this.model.router.go(`/?page=entry&id=${logId}`);
+            this.error = false;
+        } else {
+            this.error = true;
+            this.errorMessage = result.errors[0].detail;
```

Comment:
It is an array containing one or more errors, only showing the first is not that usefull I think.

Reply:
Done, mapping through the errors and showing them on top now will be fixed in the next commit